### PR TITLE
side menu bold option on select

### PR DESCRIPTION
**Pull request by benbaler**

### CODE REVIEW

**Comment by benbaler** [comment]
@omrihecht i don't know how to make the icon bold on select, i pass it as component from outside.
any ideas?

File: projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.scss [hunk 1]
```diff
@@ -16,6 +16,12 @@
       }
     }
 
+    &.selected {
+      .option-display-name {
+        font-weight: bold;
```
Comment (danazelniker) [inline projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.scss]: we don't want to add it in bob style?

**Comment by danazelniker** [comment]
@omrihecht My bad... it's in bob-style... too many branches I guess 🤦‍♀️

File: projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.scss (hunk 1 above)
Comment (omrihecht) [inline projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.scss]: why was this merged? we said we are going to do this in bob-style

**Pull request merged**